core/renderers/common/renderer.js

```diff
@@ -41,6 +39,7 @@ const object = goog.require('Blockly.utils.object');
 const {Block} = goog.requireType('Blockly.Block');
 /* eslint-disable-next-line no-unused-vars */
 const {BlockSvg} = goog.requireType('Blockly.BlockSvg');
+const {Marker} = goog.requireType('Blockly.Marker');
```

Comment:
This needs the linter disabled. Same for each of the following files. Please doublecheck there are no lint warnings before merging :)